changelog.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
+- Add mix array type support.
```

Comment:
We need to tag the issue link here

Reply:
corrected in https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/28/commits/dfabbaa76238f571c2ef9a58d207342c9b06bdef